internal/errcheck/errcheck_test.go

```diff
@@ -83,6 +88,89 @@ func TestAll(t *testing.T) {
 	test(t, CheckAsserts|CheckBlank)
 }
 
+const (
+	testVendorMain = `
+	package main
+
+	import "github.com/testlog"
+
+	func main() {
+		// returns an error that is not checked
+		testlog.Info()
+	}`
+	testLog = `
+	package testlog
+
+	func Info() error {
+		return nil
+	}`
+)
+
+func TestIgnore(t *testing.T) {
+	// copy testvendor directory into current directory for test
+	if err := os.MkdirAll("testvendor", 0755); err != nil {
```

Comment:
Writing out files directly in test because it's not possible to test vendor behavior within the `testdata` directory because Go hard-codes behavior that doesn't resolve vendored imports within a `testdata` directory.